build_tools/kickstarts/rhel-7-cdk-vagrant.ks

```diff
@@ -55,12 +55,9 @@ openshift2nulecule
 
 %post
 # Originally taken entirely from original upstream Atomic project kickstart
-
-# TODO: Remove this as soon as BZ1281805 is resolved in the official 7.2 package set
-# It is still not resolved
-rpm -e docker-selinux
-rpm -Uvh http://download.eng.pnq.redhat.com/brewroot/packages/docker/1.9.1/25.el7/x86_64/docker-selinux-1.9.1-25.el7.x86_64.rpm
-
+# Workaround for #1336857
+chcon -t docker_exec_t /usr/bin/docker-current
+chcon -t docker_exec_t /usr/bin/docker-storage-setup
```

Comment:
With latest release of docker-1.9.1-40.el7.x86_64, it should have that. Please check https://bugzilla.redhat.com/show_bug.cgi?id=1336857